plugins/rikki/heroeslounge/components/managematches/default.htm

```diff
@@ -3,8 +3,10 @@
 {% for match in __SELF__.matches %} {% if loop.index0 is even %}
 <div class="row mb-3">
     {% endif %} 
-    {% if match.wbp == null %} 
-    {% component 'scheduleMatch' class="mb-3"  id=match.id %} 
+    {% if match.wbp == null %}
+    {% if __SELF__.user.sloth.is_captain or __SELF__.user.sloth.is_divs_captain %}
```

Comment:
Please check that the sloth is also the captain of the team and not just captain of any team

Reply:
Ha, good call.  There should be a method on the sloth model `isCaptainOf(team_id)`

Reply:
Would there be a preference for determining the captain through the team model vs sloth model? As they both have a method. I was personally leaning towards team model but I'm not sure.

Reply:
Sloth model would be more natural for me, also possibly faster